CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/grafana-role/tree/develop)
 
+## [2.0.0](https://github.com/idealista/grafana_role/tree/2.0.0) (2022-11-16)
+## [Full Changelog](https://github.com/idealista/grafana_role/compare/1.6.0...2.0.0)
```

Comment:
What are the breaking changes? maybe I would add a warning to them

Reply:
Added warnings in CHANGELOG.md